seumich/models.py

```diff
@@ -386,8 +391,9 @@ def __str__(self):
     def formatted_grader_comment(self):
         if not self.grader_comment:
             return "None"
-        nl_literal_pattern = re.compile(r"\\n")
-        comment_with_repls = nl_literal_pattern.sub("<br><br />", self.grader_comment.strip())
+        comment_with_repls = NL_LITERAL_PATTERN.sub("<br><br />", self.grader_comment.strip())
```

Comment:
Is there some reason we're using .sub over a simpler .replace here?

Reply:
Fixed! Thanks, this is a simpler change.